app/assets/js/app/modules/timeout.js

```diff
@@ -124,7 +124,10 @@ domready(() => {
     let countDown = timeoutUI.onTick()
     if (countDown < 1) {
       window.clearInterval(this)
-      window.location = sessionExpiredUrl
+      fetch(sessionExpiredUrl, { method: 'POST' })
+        .then(() => {
+          window.location = sessionExpiredUrl
```

Comment:
We should use two URLs - one for expiring the session and another for displaying the session expired page.